tests/unit_tests/simtel/test_simtel_histogram.py

```diff
@@ -173,6 +173,22 @@ def test_compute_system_trigger_rate_and_table(simtel_array_histogram_instance):
     assert table["Energy (TeV)"].value[0] == 1.00000000e-03
 
 
+def test_compute_system_trigger_rate_with_input(simtel_array_histogram_instance):
+    from astropy import units as u
```

Comment:
Any particular reason why doing the import here and not as usual at the top of the test module?

Reply:
This was to avoid having a very long import at the beginning but since many of the functions anyways need the module, I now put it on top